Sources/SwiftDocC/Infrastructure/Bundle Assets/DataAssetManager.swift

```diff
@@ -210,6 +229,21 @@ public struct DataAsset: Codable {
     
 }
 
+extension DataAsset {
+    /// Metadata specific to this data asset.
+    public struct Metadata: Codable, Equatable {
```

Comment:
I'm trying to imagine how we would grow/scale/expand this. Do we know of any other metadata that we plan on adding in the future? Is there going to be groups of applicable metadata per file type?

Reply:
I don't know of any at the moment. Maybe image size would be a reasonable addition in the future? Or maybe an image could be associated with a device frame via metadata?

In the actual JSON model I tried to make this as lightweight as possible (just an additional property in the variant), but here it seemed simpler and more ergonomic to introduce a wrapper struct.